code/modules/reagents/Chemistry-Reagents/acid.dm

```diff
@@ -171,7 +171,7 @@
 	color = NECROMORPH_ACID_COLOR
 	icon = 'icons/effects/effects.dmi'
 	icon_state = "spill"
-	layer = BLOOD_LAYER	//Its a liquid, close enough
+	layer = ABOVE_OBJ_LAYER	//Ensure this draws over corruption.
```

Comment:
this is a liquid that lies on the floor, it shouldn't draw over objects.

Reply:
It's not drawn over corruption though, resulting in some complaints.

Reply:
Except I've seen a lot of people complain about it not drawing over corruption though.

Reply:
thats an unfortunate thing, but liquids don't go over corruption, and can't without screwing up the whole system. Layering is somewhat inflexible in this regard, but these two values have been carefully tuned and brook no improvement

Reply:
I'll revert this back to the blood layer then. Unfortunate, but I'm marking the issue a non-issue then. People will just have to watch out.